hail/src/main/scala/is/hail/variant/ReferenceGenome.scala

```diff
@@ -434,6 +427,24 @@ case class ReferenceGenome(name: String, contigs: Array[String], lengths: Map[St
     lo.queryInterval(interval, minMatch)
   }
 
+  def heal(tmpdir: String, fs: FS): Unit = this.synchronized {
+    // add liftovers
+    liftoverMaps = Map.empty
```

Comment:
This looks like it blows away the existing map -- can we make the liftoverMaps mutable and just add chain files that aren't in it already? It looks like if we currently do:

```
addLiftover(ctx, file2, rg2)
addLiftover(ctx, file3, rg3)
addLiftover(ctx, file4, rg4)
```

We create the first liftover object 3 times.

Reply:
added a check to make sure we aren't adding the same liftover.